cfgov/core/views.py

```diff
@@ -123,3 +126,20 @@ def submit_comment(data):
                              })
 
     return response
+
+
+@csrf_exempt
```

Comment:
would it be cleaner/nicer to use django rest framework here?  i noticed you already have it in your requirements.